reanalysis/hail_filter_and_label.py

```diff
@@ -698,8 +698,14 @@ def main(
     :param config_path: path to the config json
     :param plink_file: pedigree filepath in PLINK format
     :param out_vcf: path to write the VCF out to
+    :param tmp_path: temporary checkpoint write path
     """
 
+    # when we checkpoint to a location, we are now reading from it
+    # to maintain multiple checkpoints, we rotate between two ext.
+    checkpoint_extension = ''
+    checkpoint_flip_flop = {'': '2', '2': ''}
```

Comment:
Since you're writing to a temp location anyway, I wouldn't be too worried about the flip-flopping to save storage. It can actually be kind of helpful for debugging afterwards if you have snapshots of the various stages available.

For example our temp buckets clear out any data after 7 days, so it's just a short-term extra cost to keep everything around.

Reply:
I'll revert this and just increment for each checkpoint!